src/store/modules/contact.ts

```diff
@@ -0,0 +1,64 @@
+import { ActionContext } from "vuex";
+import { State as RootState } from "@/store/index";
+import Contact from "@/services/api/v2/contact.service";
+import axios from "axios";
+import CredentialType from "@/services/api/v2/credential-type.service";
+import { IApiPagedResult } from "@/interfaces/api/result.interface";
+import { ICredentialType } from "@/interfaces/api/v2/credential-type.interface";
+import { defaultPageResult } from "@/utils/result";
+import Http, { HttpResponse } from "@/services/http.service";
+
+const contactService = new Contact();
+const httpService = new Http();
+
+export interface ContactRequest {
+    reason: string;
+    from_name: string;
+    from_email: string;
+    comments: string;
+}
+
+export interface IncorrectInfoContactRequest extends ContactRequest {
+    reason: string;
+    error: string;
+    identifier: string;
+}
+
+export interface State {
+    requestTypes: Array<string>;
+  }
+  
+  const state: State = {
+    requestTypes: [],
+  };
+  
+  const getters = {
+    requestTypes: (state: State): Array<string> => state.requestTypes,
+  };
+  
+  const actions = {
+      async fetchRequestTypes({ commit }: ActionContext<State, RootState>, url:string): Promise<void> {
+        await Http.get<HttpResponse<string>>(
```

Comment:
You likely won't be fetching this data from the API. Can import it directly from a client side enum

Reply:
changed to get data from enum